File: examples/性能测试/函数调用数学计算.py
import time
import math

t = time.perf_counter()
result = 0.0
for i in range(1, 50000001):
    result += math.sin(i) * math.cos(i)
    result += math.sqrt(i)
    result += math.log(i)
print(result)
t1 = time.perf_counter()
print(f"{(t1 - t) * 1000:.0f}ms")

File: examples/性能测试/函数调用数学计算.lua
local t = os.clock() * 1000
local result = 0.0
for i = 1, 50000000 do
    result = result + math.sin(i) * math.cos(i)
    result = result + math.sqrt(i)
    result = result + math.log(i)
end
print(result)
local t1 = os.clock() * 1000
print((t1 - t) .. "ms")
io.read()